scripts/exglobal_atmos_vminmon.sh

```diff
@@ -44,15 +44,15 @@ if [[ -s ${gsistat} ]]; then
    #------------------------------------------------------------------
    #   Run the child sccripts.
    #------------------------------------------------------------------
-   "${USHgfs}/minmon_xtrct_costs.pl" "${MINMON_SUFFIX}" "${PDY}" "${cyc}" "${gsistat}" dummy
```

Comment:
Why is "dummy" being removed in this script?

Reply:
`dummy` was removed because `${USHgfs}/minmon_xtrct_costs.pl` only requires 4 arguments now.

Reply:
Gotcha, thanks!